module/behaviour/strategy/KeyboardWalk/src/KeyboardWalk.hpp

```diff
@@ -73,7 +73,7 @@ namespace module::behaviour::strategy {
         void look_up();
         void look_down();
         void walk_toggle();
-        void quit();
+        static void quit();
```

Comment:
Generally, when I see clang-tidy suggest this, I move the function into the cpp file as a raw function. It clearly didn't need to be on the class